tools/README.md

```diff
@@ -0,0 +1,31 @@
+## Tools
+
+## `generateImageAssets`
+
+Automatically updates your project dependencies to use a consistent [`react-native-bootsplash`](https://github.com/zoontek/react-native-bootsplash) compatible icon.
+
+** Be sure to commit your work before executing! **
+
+### Usage
+
+```bash
+node ./generateImageAssets.js --projectDir <path/to/project> --icon <path/to/icon>
```

Comment:
It could be really cool to create a bin for this script (`generate-bootsplash-assets --projectDir <path/to/project> --icon <path/to/icon>` - to allow being run from `package.json` scripts or with `npx`) https://medium.com/netscape/a-guide-to-create-a-nodejs-command-line-package-c2166ad0452e

Reply:
👍